### PR TITLE
base_optional_quick_create: port to v8 + PEP-8 compliance

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

File: base_optional_quick_create/model.py [hunk 1]
```diff
@@ -33,7 +32,9 @@ class ir_model(orm.Model):
 
     def _wrap_name_create(self, old_create, model):
         def wrapper(cr, uid, name, context=None):
-            raise orm.except_orm(_('Error'), _("Can't create quickly. Opening create form"))
+            raise orm.except_orm(
+                _('Error'),
+                _("Can't create quickly. Opening create form"))
```
Comment (pedrobaeza) [inline base_optional_quick_create/model.py:37]: This should be something like `Can't create quickly. Please open create form`

**Comment by pedrobaeza** [comment]
The rest LGTM.

**Comment by alexis-via** [comment]
@pedrobaeza 
In fact, when you activate the option on a model, the regular form view will open automatically and you never see the error message.

**Comment by pedrobaeza** [comment]
OK, then :+1:

**Comment by guewen** [comment]
:+1:

**Comment by tafaRU** [comment]
Code review and test so for me is :+1:

**Comment by tafaRU** [comment]
This PR receveid 3 approves so it can be merged :white_check_mark:

**Pull request merged**